guides/common/modules/proc_generating-metadata-and-listing-files-for-syncable-exports.adoc

```diff
@@ -0,0 +1,26 @@
+[id="generating-metadata-and-listing-files-for-syncable-exports_{context}"]
```

Comment:
```suggestion
[id="generating-metadata-and-listing-files-for-syncable-exports"]
```

suggestion: drop attribute in anchor.

For newly added files/anchors, we try to omit the "{context}" attribute unless required.